.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "themes/inventory"]
 	path = themes/inventory
-	url = https://github.com/unicef/inventory-hugo-theme.git
+	url = https://github.com/Idadelveloper/inventory-hugo-theme.git
```

Comment:
It is confusing, but you actually do not need to update the submodule link for the Pull Request! This is what I do when I am testing changes locally, but I am careful not to commit changes to the `.gitmodules` file. At the end of the day, when the CI pipeline deploys the Inventory site, we still want to pull changes from the UNICEF fork of the theme repository.

Please revert the `.gitmodules` change back to the original. Thanks!

Reply:
Okay. Will do this